src/pages/NewChatConfirmPage.tsx

```diff
@@ -144,9 +144,9 @@ function NewChatConfirmPage({newGroupDraft, allPersonalDetails}: NewChatConfirmP
                     sections={[{title: translate('common.members'), data: sections}]}
                     ListItem={InviteMemberListItem}
                     onSelectRow={unselectOption}
-                    showConfirmButton={selectedOptions.length > 1}
+                    showConfirmButton={selectedOptions.length > 0}
                     confirmButtonText={translate('newChatPage.startGroup')}
-                    onConfirm={selectedOptions.length > 1 ? createGroup : undefined}
+                    onConfirm={selectedOptions.length > 0 ? createGroup : undefined}
```

Comment:
Similarly, Can we do the following change?
```suggestion
                    onConfirm={createGroup}
```